Doc/using/mac.rst

```diff
@@ -114,15 +114,20 @@ Close this terminal window and the installer window.
 
 A default install will include:
 
-* A |usemac_applications_folder_name| folder in your :file:`Applications` folder. In here
-  you find :program:`IDLE`, the development environment that is a standard part of official
-  Python distributions; and :program:`Python Launcher`, which handles double-clicking Python
-  scripts from the macOS `Finder <https://support.apple.com/en-us/HT201732>`_.
-
-* A framework :file:`/Library/Frameworks/Python.framework`, which includes the
-  Python executable and libraries. The installer adds this location to your shell
-  path. To uninstall Python, you can remove these three things.
-  Symlinks to the Python executable are placed in :file:`/usr/local/bin/`.
+* A |usemac_applications_folder_name| folder in your :file:`Applications` folder;
+
+* Within the |usemac_applications_folder_name| folder: :program:`IDLE`, the development environment that is a standard part of official Python distributions;
```

Comment:
```suggestion
    * Within the |usemac_applications_folder_name| folder: :program:`IDLE`, the development environment that is a standard part of official Python distributions;
```

Maybe this should be made a sub bullet point rather than standalone as it refers to the previous one?